Makefile

```diff
@@ -41,6 +39,10 @@ endif
 	@./scripts/verify-gopkg.sh
 	@go test -v $(shell go list ./... | grep -v '/vendor/\|/e2e')
 
+gofmt:
+	gofmt -s -w $(GOFILES)
```

Comment:
We might be able to replace this with just `go fmt ./...`

Reply:
It automatically does as of 1.9 :)

See https://golang.org/doc/go1.9#vendor-dotdotdot

Reply:
> It automatically does as of 1.9 :)

Are we comfortable making that our minimum release, as documented [here][1]?  I see we dropped 1.8 from Travis in b64b287dfe643 (#919).

[1]: https://github.com/kubernetes-incubator/bootkube/blob/bff16e2afc34df7469e7a108cf5d87098809249a/Documentation/development.md#requirements

Reply:
> We might be able to replace this with just `go fmt ./...`

I was taking another look at this now that #955 has bumped our minimum Go version.  `go fmt …` calls `gofmt` with `-l -w` by default, which means we'd be dropping the `-s` we're currently using.  Is that acceptable?  Or do you want to keep the current, direct `gofmt` invocation?